src/MoBi.Assets/AppConstants.cs

```diff
@@ -1865,6 +1866,19 @@ public static string SelectBuildingBlocksToCloneFrom(Module module)
          {
             return $"Select building blocks to clone from {module.Name}";
          }
+
+         public static string AlsoImportIndividualsAndExpressions(int numberOfIndividuals, int numberOfExpressions)
+         {
+            var individual = "individual".PluralizeIf(numberOfIndividuals);
+            var expression = "expressions".PluralizeIf(numberOfExpressions);
+               
+            if (numberOfExpressions == 0)
+               return $"Also import {numberOfIndividuals} {individual}?";
+            else if(numberOfIndividuals == 0)
```

Comment:
no need for else here

Reply:
also feels like your text should be  $"{{numberOfIndividuals} individual".PluralizeIf(numberOfIndividuals);" or sthg lije this. Same with expression especially because you are declaring a local variable